camunda.Dockerfile

```diff
@@ -0,0 +1,165 @@
+FROM ${DIST} as dist
+
+### Application Image ###
+# https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope
+# hadolint ignore=DL3006
+FROM java as app
+# leave unset to use the default value at the top of the file
+ARG BASE_IMAGE
+ARG BASE_DIGEST
+ARG VERSION=""
+ARG DATE=""
+ARG REVISION=""
+
+# OCI labels: https://github.com/opencontainers/image-spec/blob/main/annotations.md
+LABEL org.opencontainers.image.base.digest="${BASE_DIGEST}"
+LABEL org.opencontainers.image.base.name="docker.io/library/${BASE_IMAGE}"
+LABEL org.opencontainers.image.created="${DATE}"
+LABEL org.opencontainers.image.authors="zeebe@camunda.com"
+LABEL org.opencontainers.image.url="https://zeebe.io"
+LABEL org.opencontainers.image.documentation="https://docs.camunda.io/docs/self-managed/zeebe-deployment/"
+LABEL org.opencontainers.image.source="https://github.com/camunda/camunda"
+LABEL org.opencontainers.image.version="${VERSION}"
+# According to https://github.com/opencontainers/image-spec/blob/main/annotations.md#pre-defined-annotation-keys
+# and given we set the base.name and base.digest, we reference the manifest of the base image here
+LABEL org.opencontainers.image.ref.name="${BASE_IMAGE}"
+LABEL org.opencontainers.image.revision="${REVISION}"
+LABEL org.opencontainers.image.vendor="Camunda Services GmbH"
+LABEL org.opencontainers.image.licenses="(Apache-2.0 AND LicenseRef-Zeebe-Community-1.1)"
+LABEL org.opencontainers.image.title="Camunda 8"
+LABEL org.opencontainers.image.description="Workflow engine for microservice orchestration"
+
+# OpenShift labels: https://docs.openshift.com/container-platform/4.10/openshift_images/create-images.html#defining-image-metadata
+LABEL io.openshift.tags="bpmn,orchestration,workflow"
+LABEL io.k8s.description="Workflow engine for microservice orchestration"
+LABEL io.openshift.non-scalable="false"
+LABEL io.openshift.min-memory="512Mi"
+LABEL io.openshift.min-cpu="1"
+
+ENV CAMUNDA_HOME=/usr/local/camunda
+ENV PATH "${CAMUNDA_HOME}/bin:${PATH}"
+# Disable RocksDB runtime check for musl, which launches `ldd` as a shell process
+# We know there's no need to check for musl on this image
+ENV ROCKSDB_MUSL_LIBC=false
+
+WORKDIR ${CAMUNDA_HOME}
+EXPOSE 8080 26500 26501 26502
+VOLUME /tmp
+VOLUME ${CAMUNDA_HOME}/data
+VOLUME ${CAMUNDA_HOME}/logs
+
+RUN groupadd --gid 1001 camunda && \
+    adduser --system --gid 1001 --uid 1001 --home ${CAMUNDA_HOME} camunda && \
+    chmod g=u /etc/passwd && \
+    # These directories are to be mounted by users, eagerly creating them and setting ownership
+    # helps to avoid potential permission issues due to default volume ownership.
+    mkdir ${CAMUNDA_HOME}/data && \
+    mkdir ${CAMUNDA_HOME}/logs && \
+    chown -R 1001:0 ${CAMUNDA_HOME} && \
+    chmod -R 0775 ${CAMUNDA_HOME}
+
+COPY --from=dist --chown=1001:0 /camunda/camunda-zeebe ${CAMUNDA_HOME}
+
+USER 1001:1001
+
+ENTRYPOINT ["tini", "--", "/usr/local/camunda/bin/camunda"]
```

Comment:
The Zeebe `Dockerfile` uses a dedicated [`startup.sh`](https://github.com/camunda/camunda/blob/f073cc4c0289129c8d0b08a1aa71455d3f4ecb2b/zeebe/docker/utils/startup.sh) to pass the host. Do we need this here as well?

(Tagging @Zelldon because we talked about this last week)

Reply:
@houssain-barouni, for transparency: https://app.slack.com/client/T0PM0P1SA/C037RS2JHB8

Reply:
Good point! I had in mind that [startup.sh](https://github.com/camunda/camunda/blob/f073cc4c0289129c8d0b08a1aa71455d3f4ecb2b/zeebe/docker/utils/startup.sh) just dispatches between gateway, restore, broker entrypoints, but it is doing more than that.

Reply:
it seems we are planning to get rid of the startup.sh https://github.com/camunda/camunda/issues/19114
@Zelldon , maybe we can sync on this with @npepinpe ?

Reply:
I will pair on this with Nicolas on friday

Reply:
@Zelldon and @houssain-barouni, do you think this script is a blocking topic? Basically, what I mean, can this PR still be merged without the usage of that script (or any other solution that you come up with tomorrow)?

Reply:
I guess it will also work if we use the script right?

Reply:
I think so

Reply:
Ah no, not really because it will no longer call `broker` or `gateway`. We would need another script here.

Reply:
... or extend the existing script.

Reply:
Yeah that was what I was thinking, extend to make progress but also investigate whether we can get rid of it.

Reply:
I prefer keeping it this way. It works for a simple configuration (Zeebe not running in cluster mode). Then depending on the investigation outcomes, we can adapt it later.

Reply:
For reference: https://github.com/camunda/camunda/pull/19228

However, to conclude this thread, we agreed that we should not wait for the changes, meaning
* We can merge this Dockerfile without applying a `startup.sh` script.
* We will adapt to the outcome of #19228 

@houssain-barouni, anything missing?